Extensions/PathfindingBehavior/Extension.cpp

```diff
@@ -228,6 +228,22 @@ void DeclarePathfindingBehaviorExtension(gd::PlatformExtension& extension) {
         .SetFunctionName("GetSpeed")
         .SetIncludeFile("PathfindingBehavior/PathfindingRuntimeBehavior.h");
 
+    aut.AddScopedCondition("MovementAngleIsAround",
+                          _("Angle of movement"),
+                          _("Compare the angle of displacement of an object"),
```

Comment:
Displacement => movement (I think it's more usual english) + period at the end of the sentence. Also maybe "on its path" to emphasize this is on the pathfinding path:

"Compare the angle of the movement of an object on its path."

Reply:
(I've used displacement in some part of the engine but not sure it's a good idea to surface this term to the user after thinking about it)